site/docs/reference/connectors-datacatalog/Models/fybrik-arrow-flight.md

```diff
@@ -0,0 +1,11 @@
+# fybrik-arrow-flight
+
```

Comment:
See previous comment about "fybrik"-arrow-flight and whether there is anything specific to fybrik in this connection type